heareval/predictions/task_predictions.py

```diff
@@ -79,6 +82,117 @@ def __getitem__(self, idx):
         # return self.embedding_memmap[idx], self.labels[idx]
 
 
```

Comment:
On the one hand, I'm not sure we want to do this here. On the other hand, I have no idea where we want to do this? So I guess this is fine :)